longhaul/longhaul.md

```diff
@@ -0,0 +1,235 @@
+# IoT SDK Long Haul Technical Spec
+
+The Longhaul tests are to ensure the SDK can handle the rigors of in-field life.
+
+## Delivery scheduled
+
+* Longhaul tests will follow a crawl-\>walk-\>run approach.  
+
+* onghaul tests will sim-ship tests for both node and python using the Horton framework.
+
+## Parameters for Longhaul Testing
+
+* The test should run between 12 and 336 hours (2 weeks), but could be run for much longer periods
+
+* The test will perform a variety of operations and verify the results.  For example, D2C telemetry test will verify that the telemetry is sent successfully and also  arrived at the EventHub
+
+### Crawl
+
+* Tests will send D2C messages at a fixed cadence and verify receipt
+
+* Tests will use those same D2C to deliver time-series telemetry (such as memory use, ) to the hub for dashboarding
```

Comment:
what environment/device setup will be used for running these tests? should we specify different setups (simulated device, physical devices, edge modules, etc.) that these tests should run on?